september-util/src/main/java/edu/asu/diging/gilesecosystem/septemberutil/service/impl/SystemMessageHandler.java

```diff
@@ -49,7 +49,7 @@ public SystemMessageHandler(String applicationId) {
     }
 
     @Override
-    public void handleError(String msg, Exception exception) {
+    public void handleException(String msg, Exception exception, String messageType) {
```

Comment:
@jdamerow we have ISystemMessageRequest defining ExceptionTypes values as final static string. Do we need to define extra enum? We are using SystemMessageRequest in this method so we can also use exceptiontype from same interface.